src/components/backend-ai-storage-list.ts

```diff
@@ -1098,29 +1098,46 @@ export default class BackendAiStorageList extends BackendAIPage {
     const selectNodeList = this.shadowRoot?.querySelectorAll(
       '#modify-permission-dialog mwc-select',
     );
+    const leaveList: Array<{
+      shared_user_uuid: string;
+      vfolder: string;
+    }> = [];
     const inputList = Array.prototype.filter
-      .call(
-        selectNodeList,
-        (pulldown, idx) =>
-          pulldown.value !== (this.invitees as inviteeData[])[idx].perm,
-      )
+      .call(selectNodeList, (pulldown, idx) => {
+        if (pulldown.value === 'kickout') {
+          leaveList.push({
+            shared_user_uuid: this.invitees[idx].shared_to.uuid,
+            vfolder: this.invitees[idx].vfolder_name,
+          });
+          return false;
+        }
+        return pulldown.value !== (this.invitees as inviteeData[])[idx].perm;
+      })
       .map((pulldown, idx) => ({
-        perm: pulldown.value === 'kickout' ? null : pulldown.value,
+        perm: pulldown.value,
         user: this.invitees[idx].shared_to.uuid,
         vfolder: this.invitees[idx].vfolder_id,
       }));
     const promiseArray = inputList.map((input) =>
       globalThis.backendaiclient.vfolder.modify_invitee_permission(input),
     );
-    Promise.all(promiseArray).then((response: any) => {
-      if (response.length === 0) {
-        this.notification.text = _text('data.permission.NoChanges');
-      } else {
-        this.notification.text = _text('data.permission.PermissionModified');
-      }
-      this.notification.show();
-      this.modifyPermissionDialog.hide();
-    });
+    const leavePromiseArray = leaveList.map((input) =>
+      globalThis.backendaiclient.vfolder.leave(
```

Comment:
Please check if the `/leave` API is for kicking out. I received an error message as follows:
```
{
    "type": "https://api.backend.ai/probs/insufficient-privilege",
    "title": "Insufficient privilege.",
    "msg": "Insufficient permission."
}
```